stories/components/Modal.stories.mdx

```diff
@@ -117,25 +179,16 @@ export const WithPlaygroundTemplate = ({ ...args }) => (
     </section>
 )
 
-export const ModalTemplate = ({
-    body = '',
-    title,
-    subtitle = '',
-    buttonTitle,
-    buttonText,
-    medium = false,
-    plain = false,
-    large = false,
-}) => {
+export const ModalTemplate = (args) => {
```

Comment:
Can we destructure this and group each components' props together, rather than passing everything down into each one?

Reply:
nit: Can we use function declarations for these since it's the convention in our projects?

Reply:
@frankieyan I've destructured props in the latest commit but not quite sure what you meant by grouping them together?

Reply:
The change LGTM @nats12 👍